### DISCUSSION TITLE
[Ideas]: Split project

### DISCUSSION THREAD

**Discussion by nonzzz** [nonzzz/vite-plugin-compression#53] [General]
#TL;DR

Now us plugin is stable, and its stable for a long time. And we have two function. One is `compression` the other is `tar`. FWIW, To be honest, the project is stable and i have no plan to add some features, only  to fix bugs and optimize logic. So that in my option, split this project can be better.

## Why

Like #51,  There is no problem with compress, just tar archive have some problem. So i have had to publish a new version. And is just an additional function, so i think create a new thang is better. AFAIK, Only a few people use this function.

## Context

For split project, my idea is 

- Create a new vite's plugin `vite-plugin-tar`.
- Maintaining existing code (expose the tar function from `vite-plugin-tar`)
- Create a new based on the memory tar archive basic library. (`ustart`)

## Others

I notice someone care the difference platform binary equal. If you have any ideas. Tell me pls.
